Assets/MixedRealityToolkit.SDK/Experimental/Features/UX/KeyboardInputFieldBase.cs

```diff
@@ -0,0 +1,74 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+using UnityEngine.UI;
+#if WINDOWS_UWP
+using UnityEngine;
+using UnityEngine.EventSystems;
+using Microsoft.MixedReality.Toolkit.Input;
+#endif
+
+namespace Microsoft.MixedReality.Toolkit.Experimental.UI
+{
+    /// <summary>
+    /// Base class explicitly launching Windows Mixed Reality's system keyboard for InputField and TMP_InputField
```

Comment:
This looks very interesting, following the guidance ["Documentation should be added in the same pull request as a fix / feature"](https://microsoft.github.io/MixedRealityToolkit-Unity/Documentation/Contributing/PullRequests.html#documentation-should-be-added-in-the-same-pull-request-as-a-fix--feature), it would be great to have documentation about how to invoke the system keyboard on input fields in `README_SystemKeyboard.md`

Reply:
I'm actually not feeling very confident that I'd write the right stuff here

Reply:
@julenka if writing the documentation is all that keeps the PR from being pulled in, I can try to write it, unless someone else wants to. I'm german you know ;)

Reply:
Hi @Alexees, I can make some time to write up some documentation to help here.